### ISSUE TITLE
Input Field Number / Stepper Combo - fix accessibility of vertical stepper layout

### ISSUE THREAD

**Issue by thrbnhrtmnn** [deven-org/boiler#1088]
### Description / User story

We got a report, that the vertical stepper layout is not accessible in the sizeVariants sm and md. The minimum click area must be 24px high or the distance to other clickable elements must be at least in a 24px radius. As long as the input can be smaller than 48 px in height, a vertical stepper layout is not possible.

This probably will also be reported by the storybook a11y addon, once the config has been updated and the rule for this has been turned on.

### Requirements / Prerequisites

- none

### Acceptance Criteria

- [ ] The input field number / stepper combo components have been updated in design to comply to the WCAG 2.2 requirements
- [ ] The input field number / stepper combo components have been updated in code to comply to the WCAG 2.2 requirements

### Additional information

- ...

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
